stan/math/prim/mat/fun/dot_product.hpp

```diff
@@ -19,8 +19,8 @@ namespace math {
  * size or if they are both not vector dimensioned.
  */
 template <int R1, int C1, int R2, int C2>
-inline double dot_product(const Eigen::Matrix<double, R1, C1>& v1,
-                          const Eigen::Matrix<double, R2, C2>& v2) {
+inline double dot_product(const Eigen::Matrix<double, R1, C1> &v1,
```

Comment:
Am I missing something or are the changes to the dot_product.hpp just changed positions of & and * and an added #include that I am not sure if it is needed.
I am guessing this file was changed severly in the process but was then reverted back? Can you revert these changes if they have no effect?

Reply:
@rok-cesnovar I believe this was some autoformatting thing, stuff like this is handled automatically...